### PR TITLE
update document to include serving infomation

### PR DESCRIPTION
**Pull request by utkobe**
Update the documents to show how to predict using trained model. Also add instructions on how to install packages using bip.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,8 +102,24 @@ image classification, just run the following to start the pipeline:
 $ pipcook run examples/pipelines/mnist-image-classification.json
 ```
 
-__NOTICE__: the last two examples are using Boa (pipcook python bridge layer). 
-Before run them, you need to setup Python environment. See [here](docs/tutorials/want-to-use-python.md) for more information
+__NOTICE__: The last two examples are using Boa (pipcook python bridge layer). Boa is not responsible for installing specific python packages currently. You can use `pipcook bip` as an alias for `pip` and install packages. To make pipeline-object-detection working, please make sure you have dependencies specified in [detectron2 installation reference][]. See [here](docs/tutorials/want-to-use-python.md) for more information about Boa.
+
+### Serving
+After the above pipeline is finished, you have already trained an awesome model and we have generated the prediction logics for you to use your model. This prediction package is stored in pipcook-output/[pipeline-id]/deploy. It's an independent npm package and can be easily integrated in your exsiting system. To just start a prediction server locally, you can run
```
Comment (yorkie) [inline README.md:108]: Add a blank line here.

File: README.md [hunk 2]
```diff
@@ -102,8 +102,25 @@ image classification, just run the following to start the pipeline:
 $ pipcook run examples/pipelines/mnist-image-classification.json
 ```
 
-__NOTICE__: the last two examples are using Boa (pipcook python bridge layer). 
-Before run them, you need to setup Python environment. See [here](docs/tutorials/want-to-use-python.md) for more information
+__NOTICE__: The last two examples are using Boa (pipcook python bridge layer). Boa is not responsible for installing specific python packages currently. You can use `pipcook bip` as an alias for `pip` and install packages. To make pipeline-object-detection working, please make sure you have dependencies specified in [detectron2 installation reference][]. See [here](docs/tutorials/want-to-use-python.md) for more information about Boa.
+
+### Serving
```
Comment (yorkie) [inline README.md:107]: The "Serving" and "Playground" in the same level, how about changing the "Playground" to "Pipeline"? And separate the `pipboard` to another section before the "Pipeline"?

**Comment by yorkie** [comment]
@utkobe The "Serving" and "Playground" in the same level, how about changing the "Playground" to "Pipeline"? And separate the pipboard to another section before the "Pipeline"?

File: docs/tutorials/want-to-use-python.md [hunk 3]
```diff
@@ -19,6 +19,13 @@ If you are using virtualenv or conda, you can just set up your system environmen
 $ export PYTHONPATH = /Users/venv/lib/python3.7/site-packages
 ```
 
+## How to install packages
```
Comment (yorkie) [inline docs/tutorials/want-to-use-python.md:22]: New line after headings :p

File: docs/tutorials/want-to-use-python.md [hunk 4]
```diff
@@ -19,6 +19,13 @@ If you are using virtualenv or conda, you can just set up your system environmen
...
+By default, Boa will install a conda virtual environment under the path of Boa package. To make it easier to install python libraries, you can run
+```sh
```
Comment (yorkie) [inline docs/tutorials/want-to-use-python.md:24]: Ditto, a new line is required :)

File: docs/tutorials/want-to-use-python.md [hunk 5]
```diff
@@ -19,6 +19,13 @@ If you are using virtualenv or conda, you can just set up your system environmen
...
+$ ./node_modules/.bin/bip install <package-name>
+``` 
+Actually `bip` is an alias of pip which points to correct python environment
```
Comment (yorkie) [inline docs/tutorials/want-to-use-python.md:27]:
```suggestion
> `bip` is an alias of pip which points to correct Python environment.
```

File: docs/README.md [hunk 6]
```diff
@@ -61,7 +61,7 @@ After building a machine learning application, if you want to learn more about P
   - [Get started with command-line tools](./tutorials/get-started-with-cli.md)
   - [Developer Guide](./devel/developer-guide.md)
   - [How to develop a plugin](./tutorials/how-to-develop-a-plugin.md)
-  - [Want to use Python?](./tutorials/want-to-use-python.md)
+  - [Boa specification (want to use Python in Node.js?)](./tutorials/want-to-use-python.md)
```
Comment (yorkie) [inline docs/README.md:64]: Why do we change the title for this?

Comment (utkobe) [reply docs/README.md:64]: To make it clear for someone who is using Boa independently

**Pull request merged**